### PR TITLE
Update banner for hackathon promotion

### PR DESCRIPTION
**Pull request by pwoznic**
Updated the banner and several topics to provide additional information about custom prompts, summarizers, and cross-links

### CODE REVIEW

File: www/docs/learn/grounded-generation/summarizers.md [hunk 1]
```diff
@@ -48,10 +48,12 @@ with these additional summarization options.
 
 ## Beta Summarizers
 
-We also have two new beta summarizers available for our users to try:
+We also have four beta summarizers available for our users to try:
 
 * **Growth and Scale:** `vectara-experimental-summary-ext-2023-10-23-small` (gpt-3.5-turbo)
 * **Scale only:** `vectara-experimental-summary-ext-2023-10-23-med` (gpt-4.0)
+* **Scale only:** `vectara-experimental-summary-ext-2023-12-11-sml` (gpt-3.5-turbo)
```
Comment (ofermend) [inline www/docs/learn/grounded-generation/summarizers.md:55]: Isn't this one growth+scale? (2023-12-11-sml)?

File: www/docs/learn/grounded-generation/summarizers.md [hunk 2]
```diff
@@ -48,10 +48,12 @@ with these additional summarization options.
...
+* **Scale only:** `vectara-experimental-summary-ext-2023-12-11-large:` (gpt-4.0)
```
Comment (ofermend) [inline www/docs/learn/grounded-generation/summarizers.md:56]: I think this one is gpt-4-turbo

**Review by ofermend — APPROVED** [review]
Made some comments; otherwise LGTM

File: www/docs/learn/grounded-generation/summarizers.md (hunk 1 above)
Comment (pwoznic) [reply www/docs/learn/grounded-generation/summarizers.md:55]: Yes, updating in next commit.

**Pull request merged**